repos/fdbt-site/src/pages/api/additionalPricingStructures.ts

```diff
@@ -0,0 +1,88 @@
+import { startCase } from 'lodash';
+import { NextApiResponse } from 'next';
+import { ADDITIONAL_PRICING_ATTRIBUTE } from '../../../src/constants/attributes';
+import { redirectTo } from '../../../src/utils/apiUtils';
+import { isCurrency } from '../../../src/utils/apiUtils/validator';
+import { updateSessionAttribute } from '../../../src/utils/sessions';
+import { AdditionalPricing, ErrorInfo, NextApiRequestWithSession } from '../../interfaces';
+
+export const checkInputIsValid = (inputtedValue: string | undefined, inputType: string): string => {
+    let error;
+
+    if (!inputtedValue) {
+        error = `Enter a value for the ${inputType}`;
+    } else if (Math.sign(Number(inputtedValue)) === -1) {
+        error = `${startCase(inputType)} cannot be a negative number`;
```

Comment:
the error this outputs is 'Pricing structure cannot be a negative number' - we're asking the user for a number of minutes, calling it Pricing structure is a little confusing. I would use the hint text 'Time allowance after first journey cannot be a negative number'.